### PR TITLE
Add Audit logs support to MpTelemetry-2.0

### PR DESCRIPTION
**Pull request by pgunapal**
- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

Part of #29229

### CODE REVIEW

**Comment by pgunapal** [comment]
#build

**Comment by pgunapal** [comment]
#build

**Comment by websterc87** [comment]
#build

**Comment by pgunapal** [comment]
RTC build passed: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_oJm_kKxLEe-5Tq7vYMk6IA&tab=com.ibm.team.build.web.ui.internal.editors.result.test.TestProvider

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/src/io/openliberty/microprofile/telemetry20/logging/internal/MpTelemetryLogMappingUtils.java [hunk 1]
```diff
@@ -222,6 +236,72 @@ private static void mapFFDCToOpenTelemetry(LogRecordBuilder builder, String even
         builder.setContext(Context.current());
     }
 
+    /**
+     * Maps the Audit log events to the OpenTelemetry Logs Data Model.
+     *
+     * @param builder   The OpenTelemetry LogRecordBuilder, which is used to construct the LogRecord.
+     * @param eventType The object originating from logging source which contains necessary fields.
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/src/io/openliberty/microprofile/telemetry20/logging/internal/MpTelemetryLogMappingUtils.java:243]: The description of `eventType` and `event` do not match.

**Review by fmhwong — APPROVED** [review]
LGTM with one minor comment

**Comment by pgunapal** [comment]
#build ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=d021c4c8-ab69-41fc-92f3-05b662582623) - :x: completed with errors/failures.)

**Pull request merged**